develop-docs/sdk/telemetry/traces/index.mdx

```diff
@@ -65,6 +72,21 @@ This Option replaces the non-standardized `tracingOrigins` option which was prev
 
 </Alert>
 
+### `strictTracePropagation`
```

Comment:
FMI, we want to consider to auto discard traces where the incoming trace contains an org that does not match the org configured in the SDK.